README.md

```diff
@@ -1,8 +1,70 @@
 # pulp-2to3-migration-test-fixtures
 
-This repository contains a mongodump archives, /var/lib/pulp/content archives, and /var/lib/published archives.
+This repository contains Pulp 2 snapshots in the form of mongodump archives, /var/lib/pulp/content
+ archives, and /var/lib/pulp/published archives to used in tests for the pulp-2to3-migration plugin.
+ 
 
-The following commands were used to generated the current snapshots. 
+### Structure
+
+There is a separate directory for each Pulp 2 snapshot.
+Each directory has to have:
+ - a mongodump
+ - a /var/lib/pulp/content snapshot
+ - a /var/lib/pulp/published snapshot
+ 
+Each directory should have a config to generate the Pulp 2 snapshot.
+It helps to regenerate it if needed and it serves as a documentation of what is present in the
+ snapshot.
+ 
+
+#### Naming conventions
+
+Name directory in the way it would be clear what is roughly there.
+Currently the name is not parsed and is not used in any scripts, however the suggested name is
+ the following: ``plugintype_state_usefulid1_usefulid2_etc``.  
+Plugintype is `file`, `rpm`, `container`, `deb`, or `mix` if there are multiple plugins in a
+ snapshot.  
+State can be `base`, `changed` or whatever you find useful. The idea is to have some reference for
+ the tests where multiple snapshots are required.
+The rest of identifiers which make it easier to figure out which snapshot is chosen can
+ be specified separated by the underscore.
+
+Examples:
+ * file_base_4repos
+ * file_changed_4repos_removal
+ * mix_base_rpm_container
+
+
+#### Config to generate a snapshot
```

Comment:
Is it worth automating the snapshot generation? or is it too much?

Reply:
Probably not.  At least, not beyond the level of a simple shell script that does it.

Reply:
it makes sense to provide a shell script

Reply:
I'll consider it, but not as  a part of this PR.